includes/EmailNotification.php

```diff
@@ -544,8 +544,8 @@ private function compose( \User $user ) {
 				'exception' => new \Exception(),
 			];
 
-			if ( \F::app()->wg->DisableOldStyleEmail ) {
-				$emailContext['issue'] = 'SOC-2290';
+			if ( !F::app()->wg->Title->isSpecial( "Upload" ) || $this->action !== "overwrite" ) {
```

Comment:
I'd prefer flipping the logic on this if statement to be positive since its easier to read and more direct (`F::app()->wg->Title->isSpecial( "Upload" ) && $this->action !=== "overwrite"`) but this is fine.